### DISCUSSION TITLE
Git Clone Impossible

### DISCUSSION THREAD

**Discussion by sr-rafaelneves** [digitalinnovationone/dio-lab-open-source#43872] [Git Bash]
Fala pessoal, tudo certinho com vocês?

Então, estou tentando clonar o projeto com o comando `git clone` [link https](https://github.com/digitalinnovationone/dio-lab-open-source/tree/main) do projeto e no momento está impossível clonar, já procurei algumas possíveis soluções com relação a aumentar o buffer  `git config --global http.postBuffer 524288000` tentar alterar o tipo do HTTP para `git config --global http.version HTTP/1.1` e depois tentar rodar o comando do clone e nada.

Deixo a seguir o problema:

![Capturar](https://github.com/user-attachments/assets/f6ef327f-f612-4f00-a4d2-7544f8010039)

**Se alguém chegar até esse meu post e puder ajuda sigo grato!**

**Comment by genildol** [comment]
Falar meu irmão, acabei de fazer o clone aqui e deu tudo certo, não tive nenhum erro não.
![Captura de tela 2024-08-10 114728](https://github.com/user-attachments/assets/999079f1-63d8-4dfe-b38e-88bb48bed676)

**Reply by sr-rafaelneves** [reply]
> Falar meu irmão, acabei de fazer o clone aqui e deu tudo certo, não tive nenhum erro não. ![Captura de tela 2024-08-10 114728](https://private-user-images.githubusercontent.com/169672735/356818752-999079f1-63d8-4dfe-b38e-88bb48bed676.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzMDE5MTgsIm5iZiI6MTcyMzMwMTYxOCwicGF0aCI6Ii8xNjk2NzI3MzUvMzU2ODE4NzUyLTk5OTA3OWYxLTYzZDgtNGRmZS1iMzhlLTg4YmI0OGJlZDY3Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQxNDUzMzhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZmNkN2YzNjlhNGU4Y2U0MGRkNzUyYTRjYzU2MmQ3ZjlkMzk3OTFkYWZiMjgwMDM2NDAwMzRiMmVkNTA2M2U1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.z_FoT2JmoJqprBBAr98ZpB8Z_LAhziKJiTbJ7c7LB7E)

Agora é que eu fiquei confuso mesmo, vou tentar reinstalar o meu git, está é a primeira vez que me deparo com este tipo de erro e nem o que eu pesquisei no IA também não deu jeito.

Mas agradeço o seu feedback!

**Reply by genildol** [reply]
Tranquilo, eu usei o terminal do VS Code quando não uso ele uso o Git Bash.

**Reply by sr-rafaelneves** [reply]
> Tranquilo, eu usei o terminal do VS Code quando não uso ele uso o Git Bash.

O pior @genildol é que ainda não arrumei solução, já desinstalei o git, já refiz instalação e nada mudou, vou continuar tentando achar a solução.

**Comment by sr-rafaelneves** [comment]
Conseguir concluir o clone do repo utilizando a opção através do ssh, em https estava impossível para mim clonar.

**Reply by genildol** [reply]
Opa, aí sim, as vezes tem coisas que nos faz quebrar a cabeça mesmo.
